### ISSUE TITLE
About the accuracy tested in LFW AgeDB-30 Cfp-fp

### ISSUE THREAD

**Issue by haolinlyou** [TreB1eN/InsightFace_Pytorch#142]
I have trained the repo with ms1m. After 30 epochs, the accuracy is 0.993、0.933、0.934，lower the pretrain model.
My hyper-parameters is:
LR  0.001
batch_size 384 (96*4)
IR-se50
Can you tell me what can i do for imporving accuracy

**Comment by ruiming46zrm** [comment]
set lr = 0.1 at the begining.

**Comment by haolinlyou** [comment]
> set lr = 0.1 at the begining.

OK Thank you for your reply.
By the way, i foud the loss is nearly 0.004 when training in epoch 30.

**Comment by 1icas** [comment]
> I have trained the repo with ms1m. After 30 epochs, the accuracy is 0.993、0.933、0.934，lower the pretrain model.
> My hyper-parameters is:
> LR 0.001
> batch_size 384 (96*4)
> IR-se50
> Can you tell me what can i do for imporving accuracy

你好，请问可以看一下您的loss下降曲线吗

**Comment by xszg007** [comment]
@lk123400 May I ask how to verify the acc on those three datasets? I trained my model but there was no acc output during training. I also could't find any usable verify program in this prj.

**Comment by Fanzhenxiong** [comment]
> 我已经用 ms1m 训练了回购。经过30个周期后，精度为0.993、0.933、0.934，降低预训练模型。我的超参数是： LR 0.001 batch_size 384 （96*4） IR-se50 你能告诉我我能做些什么来提高准确性吗
请问怎么测试lfw数据集和cpf数据集啊？运行哪个文件啊？
